src/pck/PckFile.h

```diff
@@ -16,7 +16,7 @@ namespace pcktool {
 constexpr uint32_t PCK_HEADER_MAGIC = 0x43504447;
 
 // Pck version
-constexpr int PCK_FORMAT_VERSION = 1;
+constexpr int MAX_PCK_FORMAT_VERSION = 1;
```

Comment:
The naming on this constant seems a bit off to me, but I don't really have a good suggestion to switch to... Maybe something like `MAX_SUPPORTED_PCK_VERSION`?